pkg/daemon/daemon.go

```diff
@@ -51,6 +54,7 @@ type Daemon struct {
 	mu *sync.Mutex
 }
 
+const scriptsPath = "/bindata/scripts/enable-rdma.sh"
```

Comment:
can you change the cont name to something like `EnableMlnxRdmaScript`

Reply:
we don't need to change it, because RDMA is not just for mellanox. It can work with other vandors